src/async_fuse/memfs/mod.rs

```diff
@@ -238,22 +237,8 @@ impl<M: MetaData + Send + Sync + 'static> FileSystem for MemFs<M> {
         };
         let lookup_res = self.metadata.lookup_helper(context, parent, name).await;
         match lookup_res {
-            Ok((ttl, fuse_attr, generation)) => {
-                debug!(
-                    "lookup() successfully found the node name={:?} \
-                        under parent ino={}, the attr={:?}",
-                    name, parent, &fuse_attr,
-                );
```

Comment:
You removed these debug informations, is it because `tracing` has done it? If so, shall we double check those `debug!` in `S3Metadata`, and remove unneccessary ones?

Reply:
There are too many logs to check; I only dealt with the ones I could see.

Reply:
Please create an issue to track this problem.

Reply:
#459